python/friesian/src/bigdl/friesian/feature/table.py

```diff
@@ -2095,6 +2095,75 @@ def get_vocabularies(self, columns):
                 .distinct().rdd.map(lambda row: row[col]).collect()
         return vocabularies
 
+    def sample_listwise(self, columns, num_list_per_key, num_examples_per_list, random_seed=None,
+                        replace=True):
+        """
+        Convert the FeatureTable to a listwise FeatureTable. The FeatureTable should be
+        grouped by key before calling sample_listwise.
```

Comment:
Add more explanations:
1) What do you mean by listwise FeatureTable?
2) What columns need to be "grouped by key"?

Maybe include a simple example.